src/main/java/be/syntra/devshop/DevshopBack/exceptions/UserNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package be.syntra.devshop.DevshopBack.exceptions;
+
+public class UserNotFoundException extends NotFoundException {
```

Comment:
change to extends CustomException and change to public UserNotFoundException(String message) {
        super(HttpStatus.NOT_FOUND, message);
    }

Reply:
fixed